config/config.yml

```diff
@@ -288,6 +288,8 @@ FIELDS_ALLOWING_VIDEO_EMBEDS: ["content"]
 
 FIELDS_ALLOWING_CSS: ["content", "endnotes", "notes"]
 
+BLACKLISTED_SRCS: []
```

Comment:
Should this config have a more specific name, like `BLACKLISTED_MEDIA_SRCS` or `BLACKLISTED_HTML_MEDIA_HOSTS`?

It should have a comment describing what it does as well:
> The domain blacklist for src URLs of video, audio, source, and track elements.
> This will allow us to block embeds from certain domains in case of abuse.

Reply:
+1 to `BLACKLISTED_MEDIA_SRCS`

Reply:
Or, actually, what about `BLACKLISTED_MULTIMEDIA_SRCS` since it's used by `Multimedia`?